tests/console/verify_network.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify basic network configuration
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_module_consoletest';
+use strict;
+use warnings;
+
+use testapi;
+use utils;
+use scheduler;
+use cfg_files_utils 'validate_cfg_file';
+
+sub run {
+    select_console 'root-console';
+    my ($nm, $dev, $cfg_files) = @{get_test_suite_data()->{network}}{
+        qw(network_manager device config_files)};
+
+    # check device UP (or UNKNOWN for some machines)
+    if (script_run("ip link show dev $dev | egrep 'state (UP|UNKNOWN)'")) {
+        die "Network device '$dev' should be UP (or UNKNOWN)";
```

Comment:
isn't there a way to know what we expect and include it in test_data?

Reply:
For what I could check with other teams, this is seen with PowerVM, machines with bridge or ovs interfaces. I decided to relax the expectations as we could schedule this module in other scenarios. AFIK it seems acceptable 'unknown'.